glusterd2/commands/volumes/volume-expand-txn.go

```diff
@@ -238,3 +239,85 @@ func updateVolinfoOnExpand(c transaction.TxnCtx) error {
 
 	return nil
 }
+
+func resizeLVM(c transaction.TxnCtx) error {
+	var req api.VolExpandReq
+	if err := c.Get("req", &req); err != nil {
+		return err
+	}
+
+	var volname string
+	if err := c.Get("volname", &volname); err != nil {
+		return err
+	}
+
+	volinfo, err := volume.GetVolume(volname)
+	if err != nil {
+		return err
+	}
+
+	var expansionTpSizePerBrick uint64
+	if err := c.Get("expansionTpSizePerBrick", &expansionTpSizePerBrick); err != nil {
+		return err
+	}
+
+	var expansionMetadataSizePerBrick uint64
+	if err := c.Get("expansionMetadataSizePerBrick", &expansionMetadataSizePerBrick); err != nil {
+		return err
+	}
+
+	var brickVgMapping map[string]string
+	if err := c.Get("brickVgMapping", &brickVgMapping); err != nil {
+		return err
+	}
+
+	if err := expandLocalBricks(volinfo, expansionTpSizePerBrick, expansionMetadataSizePerBrick, brickVgMapping); err != nil {
+		return err
+	}
+
+	// Update new volume size in bytes.
+	volinfo.Capacity = volinfo.Capacity + req.Size
+	// update new volinfo in txn ctx
+	if err := c.Set("volinfo", volinfo); err != nil {
+		return err
```

Comment:
you can return the error without doing any check

Reply:
Done